mmrotate/core/visualization/image.py

```diff
@@ -63,17 +63,20 @@ def imshow_det_rbboxes(img,
     assert labels.ndim == 1
     img = imread(img)
 
-    scores = bboxes[:, -1]
-    inds = scores > score_thr
-    bboxes = bboxes[inds, :]
-    labels = labels[inds]
+    if score_thr > 0:
+        assert bboxes is not None and bboxes.shape[1] == 6
```

Comment:
The assertion needs to be put out of the if branch. We could put it after `assert bboxes.ndim == 2`